.projenrc.js

```diff
@@ -15,7 +15,7 @@ const project = new JsiiProject({
   ],
 
   bundledDeps: [
-    'yaml@2.0.0-5',
+    'yaml@2.0.0-7',
```

Comment:
Upgraded to a more recent version since we are no longer blocked by supporting node 10.